.github/workflows/mutation.yml

```
@@ -26,6 +26,6 @@ jobs:
       os: >-
         ['ubuntu-latest']
       php: >-
-        ['8.1']
+        ['8.1', '8.2']
```

```suggestion
        ['8.2']
```

One version is enough.